test/core/xds/xds_bootstrap_test.cc

```diff
@@ -203,8 +217,8 @@ TEST(XdsBootstrapTest, ValidWithoutNode) {
   auto bootstrap_or = GrpcXdsBootstrap::Create(json_str);
   ASSERT_TRUE(bootstrap_or.ok()) << bootstrap_or.status();
   auto bootstrap = std::move(*bootstrap_or);
-  auto* server =
-      static_cast<const GrpcXdsBootstrap::GrpcXdsServer*>(bootstrap->server(0));
+  auto* server = static_cast<const GrpcXdsBootstrap::GrpcXdsServer*>(
```

Comment:
Please use `ElementsAre(EqXdsServer(...))`.

Same thing in all places where we're checking the server.

Reply:
Done